app/models/application.rb

```diff
@@ -5,4 +5,13 @@ class Application < ApplicationRecord
   validates :address, presence: true
   validates :description, presence: true
   validates_inclusion_of :status, in: ["In Progress", "Pending", "Accepted", "Rejected"]
+
+  def pet_names
+    pets.pluck(:name)
+  end
+
+  def pet_link(pet_name)
```

Comment:
Cool cool, I think stuff like generating the dynamic link in the view is fine, we'll just want to keep things like queries and what not in the model. This all works!